.buildkite/scripts/steps/functional/osquery_cypress.sh

```diff
@@ -16,5 +17,5 @@ cd "$XPACK_DIR"
 checks-reporter-with-killswitch "Osquery Cypress Tests" \
  node scripts/functional_tests \
    --debug --bail \
-   --kibana-install-dir "$KIBANA_BUILD_LOCATION" \
```

Comment:
Also curious why you changed this? We intentionally run E2E tests on the built asset.